projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.html

```diff
@@ -1,7 +1,7 @@
 <div
   (click)="closeOffCanvas()"
   [@fade]="opened ? 'in' : 'out'"
-  class="go-off-canvas-backdrop"
+  class="go-column go-column--100 go-off-canvas-backdrop"
```

Comment:
I'm not sure we need to apply these `go-column` classes here. Was this necessary?

Reply:
This was probably just a habit from what I've been doing on our Angular conversions. I can remove that.